playbooks/hadoop-build-arm64/run.yaml

```diff
@@ -0,0 +1,61 @@
+- hosts: all
+  become: yes
+  roles:
+    - install-openjdk
+    - install-maven
+  tasks:
+  # According to the investigation, the whole job will be sperated to 3 steps:
+  # Step1 - Try to build Hadoop according to https://github.com/apache/hadoop/blob/trunk/BUILDING.txt,
+  #         as protobuf 2.5.0 is not available in ARM64 platform, this try will fail. The test log for
+  #         This step will be saved to hadoop_build_no_patch.log.
+  # Step2 - Backport multi-arch patch to protobuf 2.5.0 source code and build protbuf 2.5.0 locally,
+  #         then try to build again, this try will also fail because there was no protoc-gen-grpc-java-1.15.1
+  #         in maven central for ARM64 platform, the log of this step will be saved to hadoop_build_patch_protbuf.log.
+  # Step3 - Download the pre-built ARM64 version of protoc-gen-grpc-java-1.15.1 and put it in the local
+  #         maven repo, then try to build agai, this try will success. The log of this try will be saved
+  #         to hadoop_build_patch_protobuf_and_grpc.log.
+    - name: Step 1 Try to build Hadoop according to https://github.com/apache/hadoop/blob/trunk/BUILDING.txt
```

Comment:
```suggestion
    - name: Step 1: Try to build Hadoop according to https://github.com/apache/hadoop/blob/trunk/BUILDING.txt
```

Reply:
Sorry, this cannot be done, I've wrote as this at the first place, but it turns out `:` has special meaning in ansible scripts so we cannot use it here.